backend/molgenis-emx2-beacon-v2/src/main/java/org/molgenis/emx2/beaconv2/endpoints/filteringterms/FilteringTerm.java

```diff
@@ -15,13 +15,13 @@ public FilteringTerm(String type, String id, String label, String scope) {
     this.type = type;
     this.id = id;
     this.label = label;
-    this.scope = scope;
+    this.scope = Character.toLowerCase(scope.charAt(0)) + scope.substring(1);
   }
 
   public FilteringTerm(String type, String id, String scope) {
     this.type = type;
     this.id = id;
-    this.scope = scope;
+    this.scope = Character.toLowerCase(scope.charAt(0)) + scope.substring(1);
```

Comment:
?

Reply:
apparently, strings in the 'scope' fields are not allowed to start with an uppercase character